app/models/force/application.rb

```diff
@@ -88,17 +88,11 @@ def to_domain
     # Salesforce field names. Then remove this function as it won't be needed.
     def to_salesforce
       salesforce_fields = super
+      add_salesforce_suffix(salesforce_fields)
+    end
 
-      # Add the "__c" suffix back onto Salesforce field names
-      field_names = salesforce_fields.keys
-      field_names.each do |field_name|
-        unless %w[Id Name].include?(field_name) || field_name.end_with?('__c')
-          salesforce_fields["#{field_name}__c"] = salesforce_fields[field_name]
-          salesforce_fields.delete(field_name)
-        end
-      end
-
-      salesforce_fields
+    def add_salesforce_suffix(salesforce_fields)
```

Comment:
It looks like you are overriding the method and then calling the original, so don't think you need to override it. Just calling it in `to_salesforce` should be enough.